### ISSUE TITLE
sunsetEnd and sunriseStart have incorrect name properties

### ISSUE THREAD

**Issue by SimonB27P** [rdmtc/node-red-contrib-sun-position#213]
In the main output object, within the times object, the sunriseStart object has the name property "sunrise" and the sunsetEnd object has the name property "sunset".  

This means that a JSONata search for the name sunrise or sunset gives two objects in response rather than one.

The name properties should be "sunrisestart" and "sunsetend" respectively

**Comment by Hypnos3** [comment]
I've thought about this several times and I would like to rename it sunriseStart / sunriseEnd / sunsetStart and sunsetEnd. The problem is that when a user updates the package he gets problems with all nodes where the properties are no longer available if hey has it been choosen.

For this reason, I already have a list of times that only occurs several times for backward compatibility.

**Comment by SimonB27P** [comment]
Thanks for the response.  Yes it's difficult.  If you fix it then you might break someone's flow (if they update).  But then new people like me can't understand why our flows aren't working they way we expect them to.

On the other hand, if someone knows what property he is looking for then he's probably looking for payload.times.sunriseStart.value in which case you wouldn't break his flow because that works fine.  It's the payload.times.sunriseStart.name property that's wrong and if you know what you are looking for then you already think you know what that is and you don't need to check it.

I only noticed it because I'm using JSONata to look up the values so I tried looking for an object where payload.times.*.name = "sunriseStart" and of course it doesn't exist but payload.times.*.name = "sunrise" gave me two answers instead of one.  I can get round it by using JSONata differently and using $lookup(msg.payload.times, $x).value where $x is "sunriseStart" so it's not the end of the world.  

It did take me a while to work out what the problem was though!
